### ISSUE TITLE
For more perfect

### ISSUE THREAD

**Issue by ngoncom** [arvi/Agila-Theme#4]
Hi Arvi!

Thank you so much for this theme, Agila Oceanic Next really impressed me.

But can you edit a bit 2 things:
- Reduce tab height
- Change scroll bar to another color (ex: dark color instead of skyblue) because skyblue too much highlight make me cant focus to code.

P/S for people: you can using font face "Monaco" 5.1 (support unicode) in here.
- http://www.gringod.com/2006/11/01/new-version-of-monaco-font/
- https://github.com/vjpr/monaco-bold (for bold style)

**Comment by arvi** [comment]
Thanks for the appreciation. I'm glad you like it. 😄  May I know what machine are you using for this theme? I've only tested Agila in OSX and Windows as I do not have a Linux Machine.

Regarding your suggestions:
- I will study the source code or maybe other existing themes source code with small tabs so I can apply that in Agila theme and maybe make it as an UI option instead of default. I really like how tabs are looking right now :p. If you know any theme that has this, it'd be great if you can share me their github source.
- I will make sure to add a darker color for scroll bar maybe by tomorrow or tonight. :) Just had some things I need to prioritize at the moment.

Have a nice day! 😀

**Comment by ngoncom** [comment]
Hi Arvi!

Happy to talk with you, I am using Sublime Text 3 on Windows 7 (32bits), here is my capture:

http://imgur.com/H9RqKPh

Thank!

**Comment by arvi** [comment]
Hey Ngon :)

Had the time tonight to start doing changes. Tab height has been reduced and scrollbars were modified to be distraction-free. Let me know what you think, hope the height is good. If it's good i'll apply the changes to all themes tomorrow. I'm sleepy 😄 

I've decided the scrollbars for all themes be in distraction-free mode by default.
However, for the tabs, you'd have to add this setup once I updated Agila Theme by tom.

`"theme_agila_compact_tab": true,`

https://www.dropbox.com/s/55cixjtzi6vzq6c/Agila-Theme-1.3-Draft.png?dl=0
